benchmark/Benchmark.py

```diff
@@ -8,40 +8,51 @@
 """
 
 
-import pandas
+try:
+	import pandas
+except ImportError as ex:
+	pandas = None
 import sys
 import warnings
 import math
 import os
 import numpy
-import matplotlib.pyplot as plt
-import seaborn
+try:
+	import matplotlib.pyplot as plt
+except ImportError as ex:
```

Comment:
The exception `ex` is not needed, right? Do not give it a name. (This applies to the other `except` statements, too.)